packages/backend/Dockerfile

```diff
@@ -48,4 +45,4 @@ USER node
 EXPOSE 7000
 
 # Run the app at container start.
-CMD ["node", "packages/backend", "--config", "app-config.yaml"]
\ No newline at end of file
+CMD ["envconsul", "-config", "config.hcl", "node", "packages/backend", "--config", "app-config.yaml"]
```

Comment:
Same here, if we are going to build locally then we no longer need to reference envconsul in the CMD directive right?

Reply:
same thing, just trying to be a little more transparent, put in the dockerfile what is actually getting run in k8s deployment. Maybe that is not the best approach.

Reply:
@quijones probably going to leave this as is unless any major concerns here.